developers/weaviate/concepts/replication-architecture/cluster-architecture.md

```diff
@@ -125,6 +125,24 @@ If the cluster size is 3 and the replication factor is also 3, then all nodes ca
 
 If the cluster size is 10 and the replication factor is 3, the 3 nodes which contain that data (collection) can serve queries, coordinated by the coordinator node. The client waits until x (the consistency level) nodes have responded.
 
+## Replica movement
```

Comment:
I wonder if some of the details in the "how-to" sections should be added here. 

For example: 
- Some motivations for wanting to move/copy replicas (I think increasing availability would be another good one)
- Operations are asynchronous 
- What happens if a replica is copied, rather than moved (just that shard will have a higher RF - although I'm a bit confused about what happens if a collection has an RF of x, and only some part of that collection has a higher RF because we've copied it)
- Another point might be that if you copy a shard and end up with an even number RF that might not be so good as Quorum is harder to get to.